### PR TITLE
first pass at implementing hxdl to h5py examples scripts for each app…

### PR DESCRIPTION
**Pull request by RussBerg**
…lication definition

### CODE REVIEW

File: autogenerated_examples/nxdl/python_scripts/h5py/ex_h5py_NXstxm.py [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+root['/untitled_entry/control'].attrs['EX_required'] = 'false'
+ 
+root['/untitled_entry'].create_dataset(name='title', data='!some char data!', maxshape=None)
+root['/untitled_entry/title'].attrs['type'] = 'NX_CHAR'
+root['/untitled_entry/title'].attrs['units'] = ''
+root['/untitled_entry/title'].attrs['EX_required'] = 'true'
+ 
+root['/untitled_entry'].create_dataset(name='start_time', data='2021-03-22T14:00:28.221954', maxshape=None)
+root['/untitled_entry/start_time'].attrs['type'] = 'NX_DATE_TIME'
+root['/untitled_entry/start_time'].attrs['units'] = ''
```
Comment (benajamin) [inline autogenerated_examples/nxdl/python_scripts/h5py/ex_h5py_NXstxm.py:63]: Does it make any sense for a time to have a unit?

Comment (benajamin) [reply autogenerated_examples/nxdl/python_scripts/h5py/ex_h5py_NXstxm.py:63]: @prjemian says that such cases should have a 'units' attribute with the value 'NX_UNITLESS'

Comment (RussBerg) [reply autogenerated_examples/nxdl/python_scripts/h5py/ex_h5py_NXstxm.py]: units attribute removed

**Comment by RussBerg** [comment]
after updating my version of cnxvalidate it is reporting more errors so it is clear this will need more work, removing this PR

**Pull request closed without merging**